server/protocols/webadmin/cli/README.md

```diff
@@ -0,0 +1,43 @@
+# Apache James Server/Webadmin command line interface
+
+## Development
+
+'Webadmin command-line interface is an upcoming replacement for the outdated, security-vulnerable JMX command-line interface. It also aims at providing a more modern and intuitive interface.
+It is written in Python using Click - a package that supports creating command-line interfaces. Further information about Click can be found [here](https://click.palletsprojects.com).
+
+
+## Run the script
+1. Enter the following command to get Click installed
+
+      ```$ pip install Click```
```

Comment:
whatever the final choice, `pip install` is a no go. GNU/linux users have package managers for decades and don't like a brittle python specific dep manager to install things anywhere outside their packaging system control.

Reply:
And you can't expect people to have python 3 and pip available